build_pearl.sh

```diff
@@ -41,7 +45,11 @@ poetry run pyinstaller \
     operate/pearl.py \
     --add-binary ${BIN_DIR}/aea_bin_x64:. \
     --add-binary ${BIN_DIR}/aea_bin_arm64:. \
-    --onefile \
-    --distpath $BIN_DIR \
-    --name pearl_$(uname -m)
+    --onedir \
+    --name pearl_${ARCH}
+
+# Копируем результат в middleware
```

Comment:
could you please comment them in english? until I can read russian good enough 😄